tests/console/hostname.pm

```diff
@@ -35,6 +35,13 @@ sub run {
     file_content_replace('/etc/sysconfig/network/dhcp', 'DHCLIENT_SET_HOSTNAME="yes"' => 'DHCLIENT_SET_HOSTNAME="no"');
 
     set_hostname(get_var('HOSTNAME', 'susetest'));
+
+    # We have to reconnect to be sure that the new hostname is used
+    unless (get_var('SET_CUSTOM_PROMPT')) {
+        send_key 'ctrl-d';
+        reset_consoles;
```

Comment:
This resets all consoles, even though we did not potentially logout on all consoles, see for example the failures at

https://openqa.opensuse.org/tests/overview?version=Staging%3AD&groupid=2&build=D.145.1&distri=opensuse

e.g. serial consoles are not logged out, and can thus no longer be used afterwards; consoletest_setup was the one to use serial console before hostname was changed